approval_frame/templates/registration/password_reset_confirm.html

```diff
@@ -0,0 +1,61 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  {% if validlink %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-6 col-md-offset-3'>
+        <legend>Set a new password</legend>
+        <form action="." method="post">
+          {% csrf_token %}
+          <div class='form-group {% if form.errors.new_password1 %} has-error {% endif %}'>
+            {% for error in form.errors.new_password1 %}
+              <label class='control-label' for='{{ form.new_password1.id_for_label }}'>
+                {{ error }}
+              </label>
+            {% endfor %}
+            <input type='password' class='form-control' id='{{ form.new_password1.id_for_label }}'
+                name='{{ form.new_password1.name }}' placeholder='{{ form.new_password1.label }}'>
+          </div>
+          <div class='form-group {% if form.errors.new_password2 %} has-error {% endif %}'>
+            {% for error in form.errors.new_password2 %}
+              <label class='control-label' for='{{ form.new_password2.id_for_label }}'>
+                {{ error }}
+              </label>
+            {% endfor %}
+            <input type='password' class='form-control' id='{{ form.new_password2.id_for_label }}'
+                name='{{ form.new_password2.name }}' placeholder='{{ form.new_password2.label }}'>
+          </div>
+          <p>
+            <button type='submit' class='btn btn-success'>Change Password</button>
+          </p>
+        </form>
+        <p class='pull-left'>
+          <a href='{% url 'approval_polls:index' %}'>Home</a>
+        </p>
+        <p class='pull-right'>
+          <a href="{% url 'auth_login' %}">Login</a>
+        </p>
+      </div>
+    </div>
+  {% else %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+        <legend>Password reset unsuccessful</legend>
+        <p>
+          The password reset link was invalid, possibly because it has already been used. 
```

Comment:
I think putting it in the present tense would be more appropriate here 'This password reset link is invalid'